pom.xml

```diff
@@ -76,6 +80,13 @@
     </build>
 
     <dependencies>
+        <dependency>
```

Comment:
bom goes in dependencyManagement

https://github.com/jenkinsci/bom#usage

Reply:
🤦‍♂️